tests/core/data/test_csv_data_node.py

```diff
@@ -225,10 +217,6 @@ def test_set_path(self):
         dn.path = "bar.csv"
         assert dn.path == "bar.csv"
 
-    def test_raise_error_when_path_not_exist(self):
```

Comment:
Can we replace this test with a test that the data node is created, the path is correct and the data node can be written?
SHould have a tessts for other types (excel, json, parquet) as well.

Reply:
I've added a test to make sure the file is written when a default_data is passed, otherwise it will not be written(is the same behavior as pickle). However, this cannot be factorize because it depends on the write method on each DataNode class.